common/display/display.cpp

```diff
@@ -244,4 +248,14 @@ void Display::SetBrightness(uint32_t red, uint32_t green, uint32_t blue) {
 bool Display::SetBroadcastRGB(const char *range_property) {
   return display_queue_->SetBroadcastRGB(range_property);
 }
+
+void Display::SetExplicitSync(bool ExplicitSyncEnabled) {
+  is_ExplicitSync_enabled_ = ExplicitSyncEnabled;
+  display_queue_->SetExplicitSync(ExplicitSyncEnabled);
```

Comment:
here u can displayqueue like now and in displayqueue u can call displayplanemanager and compositor. DisplayPlaneManager calls into displayplane. Compositor calls into GLRenderer, dont need static functions